instance/migrations/0114_auto_20190227_1534.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2019-02-27 15:34
+from __future__ import unicode_literals
+
+from django.db import migrations, models, transaction
+
+
+def forwards_func(apps, schema_editor):
+    """
+    Fills configuration_playbook_name according to the openedx_release
+    """
+    # Redefine function here, because if gets deleted in the future this migration
+    # would fail  otherwise
+    def get_base_playbook_name(openedx_release):
+        old_playbook_releases = [
+            'ginkgo',
+            'hawthorn',
+        ]
+        if any([release_name in openedx_release for release_name in old_playbook_releases]):
+            return 'playbooks/edx_sandbox.yml'
+        return 'playbooks/openedx_native.yml'
+
+    # We get the model from the versioned app registry;
+    # if we directly import it, it'll be the wrong version
+    OpenEdXInstance = apps.get_model("instance", "OpenEdXInstance")
+    db_alias = schema_editor.connection.alias
+    # We only need to update the OpenEdXInstance, because these AppServer
+    # should be immutable, and new AppServers will use the correct values from
+    # OpenEdXInstance model
+    queryset = OpenEdXInstance.objects.all()
+    # Make all saves() in a single transaction
+    with transaction.atomic():
+        for item in queryset:
```

Comment:
@giovannicimolin, can't you just use the `.update()` method on the queryset using the `F()` method to pass the value to the `get_base_playbook_name()` method?

Reply:
We can't do this because  `F()` is a operation that is performed on the database, so we can pass this values through any function. Only arithmetic operations are allowed. [ref](https://docs.djangoproject.com/en/2.1/ref/models/expressions/)